bin/crew

```diff
@@ -1844,7 +1844,7 @@ end
 def remove_command(args)
   args['<name>'].each do |name|
     search name
-    Command.remove(@pkg)
+    Command.remove(@pkg, verbose: CREW_VERBOSE, force: @opt_false)
```

Comment:
Why not simply `force: false`?

Reply:
oops, typo. that's meant to say @opt_force